### DISCUSSION TITLE
how can i modify packets?

### DISCUSSION THREAD

**Discussion by Sombian** [Minestom/Minestom#1874] [Help] [answered]
`PlayerPacketOutEvent.class` provides `getPacket` and `getPlayer()` which is useful, but how can i modify these?

**Comment by ZakShearman** [comment, marked as answer]
Minestom doesn't allow the modification of packets.

Your only options are to:
  - Modify them separately on some kind of proxy
  - Send different packets in the first place (e.g. I send my own metadata packets in some cases)
